src/js/SubmissionContainer.jsx

```diff
@@ -52,22 +53,26 @@ var SubmissionContainer = React.createClass({
       )
     }
 
-    if(code > 3) progress = <p>Progress component goes here</p>
+    if(code > 3) progress = <p className="third">Progress component goes here</p>
 
-    if(code > 6) editsContainer = <EditsContainer/>
+    if(code > 5) editsContainer = <EditsContainer/>
```

Comment:
Ah, yeah. Right.

What do you think about adding a few more states?
- `6`: Syntactical begins
- `7`: Syntactical completes
- `8`: Validity begins
- `9`: Validity completes

For each edit type.

Then, when polling, we could start to show by type?